10.1/root/usr/share/container-scripts/mysql/README.md

```diff
@@ -124,6 +124,42 @@ variable with the full path of the file you wish to use. For example, the defaul
 location is `/etc/my.cnf` but you can change it to `/etc/mysql/my.cnf` by setting
  `MYSQL_DEFAULTS_FILE=/etc/mysql/my.cnf`
 
+Upgrading and datadir checking
+------------------------------
+MySQL and MariaDB use versions that consist of three numbers (e.g. 5.6.23). When two
+versions are different only in the last part, they are compatible, but when upgrading
```

Comment:
Ah, I read this like dozen of times, ... :) and now I get it right...   I understood that statement like 'when two _numbers_ are different only in the last part', so I would suggest:
" ... that consists of three numbers `X.Y.Z` (e.g. 5.6.23).  For version changes in `Z` part, the server's binary data format stays compatible and thus no special upgrade procedure is needed.  For upgrades from `X.Y` to `X.Y+1`,  we need to consider upgrading ... "

  co